ccm_web/server/src/api/dtos/api.section.external.users.dto.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ArrayMaxSize,
+  ArrayMinSize,
+  IsArray,
```

Comment:
Please rename this file to `api.external.users.dto.ts`. Not pertaining to sections anymore.

Reply:
🏁